src/main/java/server/Server.java

```diff
@@ -0,0 +1,40 @@
+package main.java.server;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.logging.Level;
+
+class Server {
+    private final Integer port;
+    private boolean serverIsRunning = true;
```

Comment:
I wonder if this should be set to `true` by default. Seems like it could cause some weird debugging scenarios if the call to `server.start()` throws an exception 🤔